integration-tests/src/test/java/org/openmrs/module/fhir2/providers/r4/ConditionFhirResourceProviderIntegrationTest.java

```diff
@@ -353,8 +353,8 @@ public void shouldSearchForAllConditionsAsJson() throws Exception {
 	
 	@Test
 	public void shouldReturnSortedAndFilteredSearchResultsForConditionsAsJson() throws Exception {
-		MockHttpServletResponse response = get("/Condition?clinical-status=active&onset-date=2008&_sort=-onset-date")
```

Comment:
Not sure why previously this was written to include clinical status yet in versions lower than 2.2 its not possible since in versions than 2.2, the FHIR translation of the Condition was based on the OpenMRS `Obs` object which clearly didnt have clinical statuses.

Reply:
added back `Condition?clinical-status=active` since it will be ignored anyway